### DISCUSSION TITLE
Ender 7 source

### DISCUSSION THREAD

**Discussion by alonsanchez20** [CrealityOfficial/Creality_Sonic_Pad#80] [General]
Would be nice to have Ender 7 open source repo. Ender 7 users are forgotten by Creality, we dont get repos, bltouch firmware and other updates like other printers do.

**Comment by Frits183** [comment]
> I have installed Orca, but the creality ender 7 is missing from the printer list. Can someone included it to complete the series'.
> 
> Its a marlin based corexy printer, it has a buildplate from 252_252 mm and 300 high.
> 
> Thanks
> 
> **Which printers will be beneficial to this feature** E.g. Voron/Klipper based printer/etc...
> 
> **Describe the solution you'd like** A clear and concise description of what you want to happen.
> 
> **Describe alternatives you've considered** A clear and concise description of any alternative solutions or features you've considered.
> 
> **Additional context** Add any other context or screenshots about the feature request here.
> 
> A solution for "Creality Ender 7 " a Marlin based printer.
> 
> On the Creality page A Picture of the Ender 7 to choose from with start up and end G code, and the normal max speed for the printer.
> 
> As a sollution you could take a corexy marlin printer an put the picture from the Ender 7 with it. For now i use just a ordernerly Marlin Printer and adjusted it until it worked for me.
> 
> I now there aren't so much Ender 7 user's ! But it would be nice if the Creality Ender serie's is completed with the 7.
> 
> As i am a starter and just a user and not a developer i think my issue is easy to ad. ![Ender-7](https://private-user-images.githubusercontent.com/53749250/295970003-9fb5f1ee-4908-4306-927d-a8f05231d023.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUwNDg5NDMsIm5iZiI6MTcwNTA0ODY0MywicGF0aCI6Ii81Mzc0OTI1MC8yOTU5NzAwMDMtOWZiNWYxZWUtNDkwOC00MzA2LTkyN2QtYThmMDUyMzFkMDIzLmpwZWc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExMlQwODM3MjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kNTkyMjRjMDE2Y2Y1N2JmZjA4Y2Q0ZTFkZDg1NjQyODU0YzRiZDY4ZDk3ODQxYzRmMGI1NjMwNjVhODcxOTNiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.1cZVtpOBwz_DLRF-V83DB6KXD1pPLJoTJnfk5Q7yiS8)
